### ISSUE TITLE
[$1000] Dev - Getting an error while running the local web app - reported by @parasharrajat

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#11275]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. run the local web server via npm run web.
2. Notice the error on terminal or going to localhost:port.

## Expected Result:
No error should appear 

## Actual Result:
Getting an error while running the local web app

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 
**Reproducible in staging?:** need repro
**Reproducible in production?:** need repro
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
![image - 2022-09-26T003915 608](https://user-images.githubusercontent.com/43995119/192195051-d117b7e0-7703-40c8-83bd-e4253943132a.png)

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @parasharrajat
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1662401615610659

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by NicMendonca** [comment]
> run the local web server via npm run web.
I am not sure if/ how I can do this 😓 so I am going to ask for an engineering opinion!

**Comment by arosiclair** [comment]
No repro with the latest on `main` (SHA: `2bef1705d5aac60c9463cc0f080155dc9925d033`).  @parasharrajat did you `npm install`?

**Comment by arosiclair** [comment]
Ok thanks I see it now. I'm not sure of a resolution for this but I see someone just posted an issue about it in the `react-native-renanimated` repo [here](https://github.com/software-mansion/react-native-reanimated/issues/3614). From what I can tell this hasn't had much effect on NewDot dev so I'll toss this back to the pool.

cc @roryabraham @marcaaron since you were the last guys to touch `react-navigation` and `react-native-reanimated` in package.json.

**Comment by roryabraham** [comment]
Confirmed that this is reproducible: 

<img width="1787" alt="image" src="https://user-images.githubusercontent.com/47436092/196555365-dcca3794-219a-41bf-9560-0fdac72c8895.png">

**Comment by Christinadobrzyn** [comment]
@roryabraham Just catching up here, do we want this job to go external?

**Comment by roryabraham** [comment]
I'm not sure if the error is in `react-navigation` or in `reanimated`, but it's one of those (not `E/App`). I think the error is most likely actually in `react-navigation`.

**Comment by roryabraham** [comment]
With a little bit of digging it should be pretty easy to track down. Probably some API change in `Reanimated` that `react-navigation` has yet to adapt to.

**Comment by Christinadobrzyn** [comment]
Job created in Upwork: 

Internal posting: https://www.upwork.com/ab/applicants/1582963753329188864/job-details
External posting: https://www.upwork.com/jobs/~01e7b7f771b0a1a9a1

**Comment by Christinadobrzyn** [comment]
Price doubled $500

**Comment by Christinadobrzyn** [comment]
Price doubled $1000 - still waiting on proposals

**Comment by parasharrajat** [comment]
I think this is fixed.

**Comment by dangrous** [comment]
Thanks for calling that out @parasharrajat ! I also do not see this on my local.

**Comment by thesahindia** [comment]
This is what the [CONTRIBUTING.md](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#proposing-a-job-that-expensify-hasnt-posted) says:

"If the bug is fixed by a PR that is not associated with your bug report, the contributor is not eligible for compensation unless they can find the PR that fixed the bug then show their bug report preceded the one associated with the PR."

**Comment by parasharrajat** [comment]
I think that is not needed for this issue as it was clearly [reproducible](https://github.com/Expensify/App/issues/11275#issuecomment-1283069975) when created on GH and was recently got fixed with some PR. So it is clear that some PR fixed it.

**Comment by thesahindia** [comment]
> I think that is not needed for this issue as it was clearly [reproducible](https://github.com/Expensify/App/issues/11275#issuecomment-1283069975) when created on GH and was recently got fixed with some PR. So it is clear that some PR fixed it.

I think this case should be discussed on slack since finding the PR can be time consuming but I think it is also important because we can't say for sure if the PR that fixed an issue wasn't already planned.

**Comment by Christinadobrzyn** [comment]
Sorry, just to make sure, the job was fixed without hiring?

I can ask about the reporting bonus @parasharrajat, I just want to make sure there's no one else to pay for this job.

**Comment by parasharrajat** [comment]
Yup, a PR for another issue https://github.com/Expensify/App/issues/11275#issuecomment-1305795743 fixed this.

**Comment by Christinadobrzyn** [comment]
Awesome! Thanks @parasharrajat - good to go with a reporting bonus based on this [chat](https://expensify.slack.com/archives/C01SKUP7QR0/p1667979039442149).  I hired you in Upwork for this job so I can pay the reporting bonus.

**Comment by Christinadobrzyn** [comment]
Okay, paid @parasharrajat the reporting bonus and closed the job in Upwork. Closing this GH as this job is fixed but please let me know if I'm missing something
